### ISSUE TITLE
`@stylistic/linebreaks` drops every comment a selector spanning lines holds

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#269]
Found while fixing [#235](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/235), which is the same defect in the same fixer, one property over. The sweep of [#193](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/193) missed it because it looked for rules writing `raws.selector.raw`, and this one writes the bare property instead — the shape [#194](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/194) had to be set apart from that sweep for the same reason.

### What minimal example or steps are needed to reproduce the bug?

A comment inside a selector that spans lines:

```css
a /* c */,
b { c: d; }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "plugins": [
    "@stylistic/stylelint-plugin"
  ],
  "rules": {
    "@stylistic/linebreaks": "windows"
  }
}
```

### How did you run Stylelint?

```shell
stylelint "**/*.css" --config myconfig.json --fix
```

### What did you expect to happen?

Every line ending rewritten, and every other character of the file left exactly as it was.

### What actually happened?

The comment is gone, and nothing is reported:

```css
a ,
b { c: d; }
```

Under `postcss-scss` a comment written with a double slash goes the same way. Under `postcss-less` the comment survives, since that syntax keeps it in the bare selector rather than in a raw.

A selector holding no line break keeps its comments, and by accident: `fixData` hands such a selector back unchanged, so the bare property still stands for `raws.selector.value` and PostCSS goes on printing the raw. Only a selector the fix actually rewrites loses them.

### What is the cause?

The rule writes `node.selector`, the copy PostCSS hands back with every comment taken out, and PostCSS then throws `raws.selector` away — the raw the comments live in, and under `postcss-scss` the `scss` copy it prints.

```js
if (`selector` in node) node.selector = fixData(node.selector)
```

### Do you have a proposal to fix the bug?

The pair [#194](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/194) gave `indentation`, which holds it as two module-level functions of its own, `ruleSelector` and `setRuleSelector`. They are the selector's answer to `getDeclarationValue`/`setDeclarationValue` and `getAtRuleParams`/`setAtRuleParams`, and the second caller is what says they belong in `lib/utils/` beside those two pairs rather than inside one rule.
